### ISSUE TITLE
about loss in sdmaie.py

### ISSUE THREAD

**Issue by limcce** [nashrahhaque/SD-MIAE#1]
The paper say "use negative cross-entropy as the adversarial loss",but in the code just use "loss.backward",why?
